src/jit/compiler.h

```diff
@@ -7126,6 +7126,27 @@ class Compiler
         return m_promotedStructDeathVars;
     }
 
+    void ClearPromotedStructDeathVars()
+    {
+        if (m_promotedStructDeathVars != nullptr)
+        {
+            m_promotedStructDeathVars->RemoveAll();
+        }
+    }
+
+    bool LookupPromotedStructDeathVars(GenTree* tree, VARSET_TP** bits)
```

Comment:
Do we need a header here?

Reply:
I'd be in favor of minimal headers - or just somehow make it clearer that these three methods, plus `m_promotedStructDeathVars` are all addressed by the comment above. (And while you're here, I think that in the description above, where it says "If such a node is *not* mapped in this table, one may assume that all the (tracked) field vars die at this point", you could change "this point" to "that GT_OBJ node".